Dockerfile

```diff
@@ -23,16 +23,14 @@ RUN addgroup -g 1001                \
             -G gosnmp gosnmp
 
 RUN chmod -R a+rw /etc/snmp /var/lib/net-snmp/
-RUN pip install snmpsim
 
 # Copy local branch into container
 USER gosnmp
 WORKDIR /go/src/github.com/gosnmp/gosnmp
 COPY --chown=gosnmp . .
 
-RUN go get github.com/stretchr/testify/assert && \
-    make tools && \
-    make lint
+RUN go get github.com/stretchr/testify
```

Comment:
Do we still need this `go get`? Go should automatically download it based on the Go modules.